README.md

```diff
@@ -36,39 +55,39 @@ The CSV files should be located in:
 
 With the files in the right folder, you can run the import
 command (It is faster now, with postgres =D )
-```
-$ make db-seed
+
+``` sh
+make db-seed
 ```
 
-# Execution
+### 3 - Running the project
 
 _The database schema must be created before starting the application, or
 it will crash otherwise. To do so, you have two options:_
```

Comment:
Antes o readme dizia
- Run the command mentioned above, importing the data from the CSV files  
- Create the schema without any data, by running `make db-migrate`

Agora é só o make run